src/protagonist/DLCS.Repository.Tests/Messaging/EngineClientTests.cs

```diff
@@ -104,7 +104,41 @@ public async Task SynchronousIngest_CallsEngineWithCurrentModel_IfUseLegacyEngin
             {
                 ReferenceHandler = ReferenceHandler.Preserve
             });
-        body.Should().BeEquivalentTo(ingestRequest);
+        
+        body.Asset.Should().BeEquivalentTo(new Asset
```

Comment:
is it worth instead of checking everything for null, to instead just check the id's are equivalent?